js/index.js

```diff
@@ -37,6 +37,117 @@ const siteContent = {
   },
 };
 
+
+
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Updating Nav items
+let navItems = document.getElementsByTagName('a');
+navItems[0].textContent = siteContent["nav"]["nav-item-1"];
+navItems[1].textContent = siteContent["nav"]["nav-item-2"];
+navItems[2].textContent = siteContent["nav"]["nav-item-3"];
+navItems[3].textContent = siteContent["nav"]["nav-item-4"];
+navItems[4].textContent = siteContent["nav"]["nav-item-5"];
+navItems[5].textContent = siteContent["nav"]["nav-item-6"];
+
+//Stretch
+
+let greenNav = document.querySelectorAll('a');
+greenNav[0].style.color = "green";
+greenNav[1].style.color = "green";
+greenNav[2].style.color = "green";
+greenNav[3].style.color = "green";
+greenNav[4].style.color = "green";
+greenNav[5].style.color = "green";
+
+let newNav = document.querySelector("nav");
+let newNavItem = document.createTextNode('Im New');
+let newNavItem2 = document.createTextNode('Im New Too');
+newNav.appendChild(newNavItem2);
+newNav.prepend(newNavItem);
+newNav.style.color = 'green';
+
+
+
+
+//Updating cta-text...no way I could do this using the cta object
+let h1Tag = document.querySelector('h1');
+let lineBreak = document.createElement('br');
+let lineBreak2 = document.createElement('br');
```

Comment:
Oh my... I should have done this. How come I didn't think of creating a `<br>` element! This part made an entire cohort livid and seeing you solve this puzzle is amazing.